t/t3405-rebase-malformed.sh

```diff
@@ -69,4 +69,11 @@ test_expect_success 'rebase commit with diff in message' '
 	test_cmp G G0
 '
 
+test_expect_success 'rebase exec modifies rebase-todo' '
+        git rebase HEAD -x "echo x \"echo foo >H\" >> \"$(
```

Comment:
You add this test case to the "malformed" test script... Wrong script?

In any case, it would be easier to read and understand if we used something like `touch telltale` instead of the `echo` with the quoted quotes. Also, for readability, I would strongly suggest setting `todo="$(git rev-parse --git-path rebase-merge/git-rebase-todo)"` and then using that, and instead of appending to the `git-rebase-todo` script, simply replace it with a single command.